bin/node/runtime/src/lib.rs

```diff
@@ -320,9 +320,10 @@ parameter_types! {
 	pub const BondingDuration: pallet_staking::EraIndex = 24 * 28;
 	pub const SlashDeferDuration: pallet_staking::EraIndex = 24 * 7; // 1/4 the bonding duration.
 	pub const RewardCurve: &'static PiecewiseLinear<'static> = &REWARD_CURVE;
-	pub const ElectionLookahead: BlockNumber = EPOCH_DURATION_IN_BLOCKS / 4;
 	pub const MaxNominatorRewardedPerValidator: u32 = 64;
-	pub const MaxIterations: u32 = 5;
+	pub const ElectionLookahead: BlockNumber = EPOCH_DURATION_IN_BLOCKS / 4;
+	pub const MaxIterations: u32 = 10;
+	pub SolutionImprovementThreshold: Perbill = Perbill::from_parts(Perbill::from_percent(1).deconstruct() / 10);
```

Comment:
How was the number chosen? Is 0.1% really worth the processing time? Is there a way to translate that into rewards/security gains?

Reply:
> Is there a way to translate that into rewards/security gains?

@AlfonsoCev This is a cool idea but more or less beyond my immediate scope of decision. Perhaps something that you can look more into.